### PR TITLE
formatting changes to conclusion

### PR DESCRIPTION
**Pull request by preranap1**
made some formatting changes to conclusion to address issue 804

### CODE REVIEW

**Comment by harizhisham** [comment]
hey @preranap1 it says here you made changes to 3 files. Was that the case?

File: 06-conclusion.Rmd [hunk 1]
```diff
@@ -96,20 +100,17 @@ Data visualization allows decision makers to make sense out of the visible patte
...
 
-* **Share what you have found.** A visualization can be an effective influencer if used in the correct way and at the correct time. 
+* **Share what you have found** A visualization can be an effective influencer if used in the correct way and at the correct time. 
 
 
-* **Think as author / reader** When designing graphs, try to stand at the reader side, to check whether the graph is making sense or if there is any misleading information being showcased. As a reader, one needs to detect if there is any visualization deception in the graph. Before making any conclusion, understanding details and information the graph is providing like legend, caption, numbers, etc. is important. 
+* **Think as author / reader* When designing graphs, try to stand at the reader side, to check whether the graph is making sense or if there is any misleading information being showcased. As a reader, one needs to detect if there is any visualization deception in the graph. Before making any conclusion, understanding details and information the graph is providing like legend, caption, numbers, etc. is important.
```
Comment (pragyansharma24) [inline 06-conclusion.Rmd]: Hi Prerna, 
I think by mistake one * has been deleted.
The whole paragraph is bold now. Please add ** after the author/reader.

File: 06-conclusion.Rmd [hunk 2]
```diff
@@ -96,20 +100,17 @@ Data visualization allows decision makers to make sense out of the visible patte
 
 * **Deliver value in your visualization.** Many graphs may be aesthetically pleasing or exciting but deliver no value. A graph with good value should deliver important insights to aid decision-making or prompt an action. Keep in mind, the main objective of a visualization is to deliver value, and each design decision should help achieve this.
  
-* **Understanding your audience is the most difficult and important part of developing a visualization.** Different audiences will feel differently about the same chart. So when we develop the argument from the data, we should keep our intended audience in mind.
+* **Understanding your audience is the most difficult and important part of developing a visualization.* Different audiences will feel differently about the same chart. So when we develop the argument from the data, we should keep our intended audience in mind.
 
-* **Share what you have found.** A visualization can be an effective influencer if used in the correct way and at the correct time. 
+* **Share what you have found** A visualization can be an effective influencer if used in the correct way and at the correct time.
```
Comment (pragyansharma24) [inline 06-conclusion.Rmd]: Also, all the points in the bold statements end with a full stop. 
Should we keep it consistent and add (.) after bold statements?

Comment (harizhisham) [reply 06-conclusion.Rmd]: > Also, all the points in the bold statements end with a full stop.
> Should we keep it consistent and add (.) after bold statements?

I agree with this. I looked thru the other points and they all end with a period. I think we should keep the period here as well.

**Comment by mschermann** [comment]
@preranap1 Please resolve conflicts and merge

File: 06-conclusion.Rmd (hunk 1 above)
Comment (preranap1) [reply 06-conclusion.Rmd]: Hey. Thanks for pointing this out. will change this now.

File: 06-conclusion.Rmd (hunk 2 above)
Comment (preranap1) [reply 06-conclusion.Rmd]: hey guys. Thanks for pointing this out. Will get this resolved.

**Comment by preranap1** [comment]
I was unable to resolve conflicts in the data_reader.rmd file hence I have closed this branch and will upload a new branch with changes. Thanks.

**Pull request closed without merging**